package/libpiksi/libpiksi/include/libpiksi/metrics_foreach.h

```diff
@@ -23,45 +23,78 @@
+#define _M_FOREACH_19(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_18(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_20(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_19(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_21(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_20(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_22(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_21(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_23(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_22(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_24(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_23(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_25(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_24(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_26(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_25(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_27(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_26(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_28(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_27(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_29(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_28(Context, TheMacro, __VA_ARGS__)
+#define _M_FOREACH_30(Context, TheMacro, x, ...) TheMacro(Context, x) _M_FOREACH_29(Context, TheMacro, __VA_ARGS__)
 
 #define _M_FOREACH_NARG(...) _M_FOREACH_NARG_(__VA_ARGS__, _M_FOREACH_RSEQ_N())
-#define _M_FOREACH_NARG_(...) _M_FOREACH_ARG_N(__VA_ARGS__) 
-#define _M_FOREACH_ARG_N(_1, _2, _3, _4, _5, _6, _7, _8, _9, _10, _11, _12, _13, _14, _15, _16, _17, _18, _19, _20, _21, _22, _23, _24, _25, _26, _27, _28, _29, _30, N, ...) N 
-#define _M_FOREACH_RSEQ_N() 30, 29, 28, 27, 26, 25, 24, 23, 22, 21, 20, 19, 18, 17, 16, 15, 14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0
+#define _M_FOREACH_NARG_(...) _M_FOREACH_ARG_N(__VA_ARGS__)
+#define _M_FOREACH_ARG_N(_1,  \
+                         _2,  \
+                         _3,  \
+                         _4,  \
+                         _5,  \
+                         _6,  \
+                         _7,  \
+                         _8,  \
+                         _9,  \
+                         _10, \
+                         _11, \
+                         _12, \
+                         _13, \
+                         _14, \
+                         _15, \
+                         _16, \
+                         _17, \
+                         _18, \
+                         _19, \
+                         _20, \
+                         _21, \
+                         _22, \
+                         _23, \
+                         _24, \
+                         _25, \
+                         _26, \
+                         _27, \
+                         _28, \
+                         _29, \
+                         _30, \
+                         N,   \
+                         ...) \
+  N
```

Comment:
😨

Reply:
Hrmmm..... is there no way to turn off the formatter for certain files?